src/clap-saw-demo-editor.cpp

```diff
@@ -168,6 +175,14 @@ bool ClapSawDemo::guiSetParent(const clap_window *window) noexcept
     return true;
 }
 
+bool ClapSawDemo::guiSetScale(double scale) noexcept
```

Comment:
We should add a comment here as to what this does if possible?

Reply:
Hopefully useful
[11b62ee](https://github.com/surge-synthesizer/clap-saw-demo/pull/19/commits/11b62eeb61a6a1f97bc3d33aaea06e56ee49f039)